parquet-column/src/main/java/org/apache/parquet/schema/Types.java

```diff
@@ -305,6 +306,7 @@ public P named(String name) {
   public abstract static class
       BasePrimitiveBuilder<P, THIS extends BasePrimitiveBuilder<P, THIS>>
       extends Builder<THIS, P> {
+    private final Logger log = org.slf4j.LoggerFactory.getLogger(BasePrimitiveBuilder.class);
```

Comment:
This should be `static final` and I would normally add an import for `LoggerFactory`.